pkg/labels/arraylist.go

```diff
@@ -118,6 +146,20 @@ func (lsp *LabelArrayList) Merge(include ...LabelArray) LabelArrayList {
 	return lsp.MergeSorted(incl)
 }
 
+func (lsp *LabelArrayList) InsertSorted(la LabelArray) {
+	for j := 0; j < len(*lsp); j++ {
+		if la.Less((*lsp)[j]) {
+			*lsp = append((*lsp)[:j+1], (*lsp)[j:]...) // make space at merged[j]
+			(*lsp)[j] = la
```

Comment:
nit: Note that you could use `slices.Insert` here which as some fastpaths in cases where the target slice still has available capacity.

Reply:
Applied, thanks!

Reply:
Turns out this was dead code left around from development of the PR, removed.